client/src/components/ui/itemrow/AcceptRow.svelte

```diff
@@ -7,23 +7,37 @@
     import { ContextPayload, RowType, IconSize, Events } from '../../types.ts';
     import { Document } from '../../../../../model/src/document.ts';
     import { Category } from '~model/category.ts';
+    import { Snapshot } from '~model/snapshot.ts';
 
     export let iconSize: IconSize;
     export let doc: Document['id'];
     export let category: Category['name'];
     export let title: Document['title'];
+    export let creation: Snapshot['creation'];
 
     const dispatch = createEventDispatcher();
     const rowEvent: ContextPayload = {
         ty: RowType.AcceptDocument,
         id: doc,
     };
+
+    function redirectHandler() {
+        window.location.href = `/track?id=${doc}`;
+    }
+    
+    const docDisplay = `${doc.slice(0,5)}...${doc.slice(-5)}`;
 </script>
 
 <RowTemplate
-    title={`${title} ID: ${doc} Category: ${category}`}
     {iconSize}
     on:overflowClick={() => dispatch(Events.OverflowClick, rowEvent)}
+    on:rowContainerClick={ () => redirectHandler()}
```

Comment:
Don't forget about the free-function form!
```suggestion
    on:rowContainerClick={redirectHandler}
```